src/handlers/major_change.rs

```diff
@@ -173,6 +178,41 @@ async fn handle_input(
                 config.meeting_label.clone(),
             )
         }
+        Invocation::Rename => {
+            let mut zulip_topic = format!(" {}", issue.zulip_topic_reference());
+            // We prepend the issue title, truncating such that the overall length does
+            // not exceed 60 characters (a Zulip limitation).
+            zulip_topic.insert_str(
+                0,
+                &issue.title[..std::cmp::min(issue.title.len(), 60 - zulip_topic.len())],
+            );
+
+            let zulip_send_req = crate::zulip::MessageApiRequest {
+                recipient: crate::zulip::Recipient::Stream {
+                    id: config.zulip_stream,
+                    topic: &zulip_topic,
+                },
+                content: "The associated GitHub issue has been renamed. Renaming this Zulip topic.",
+            };
```

Comment:
Hmm, I think this will post a new topic with the new title rather than posting to the old topic. I guess we'll have to include the old title in `Invocation::Rename`?